Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -1,17 +1,649 @@
+            "linkTarget": "parameter",
+            "linkLabel": "Anomalies",
+            "subTarget": "2",
+            "style": "secondary"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Host Config",
+            "subTarget": "3",
+            "style": "secondary"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "testDataExist",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Tabs"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [],
+        "parameters": [
+          {
+            "id": "8a72f4ec-ce64-49b7-88dc-d9b27de74f7a",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "value": {
+              "durationMs": 86400000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 300000
+                },
+                {
+                  "durationMs": 900000
+                },
+                {
+                  "durationMs": 1800000
+                },
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                }
+              ]
+            },
+            "label": "Time range"
+          },
+          {
+            "id": "fae26122-3768-43dd-ac98-9b9768924799",
+            "version": "KqlParameterItem/1.0",
+            "name": "Granularity",
+            "type": 2,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[\r\n    { \"value\":\"1m\", \"label\":\"1 Minute\", \"selected\":false},\r\n{ \"value\":\"5m\", \"label\":\"5 Minutes\", \"selected\":false},\r\n{ \"value\":\"15m\", \"label\":\"15 Minutes\", \"selected\":true},\r\n{ \"value\":\"30m\", \"label\":\"30 Minutes\", \"selected\":false},\r\n{ \"value\":\"1h\", \"label\":\"1 Hour\", \"selected\":false}\r\n]\r\n",
+            "timeContext": {
+              "durationMs": 0
+            },
+            "timeContextFromParameter": "TimeRange"
+          },
+          {
+            "id": "8e3577a4-1f9f-4935-a2b2-c42a6911ae8a",
+            "version": "KqlParameterItem/1.0",
+            "name": "Host",
```

Comment:
this param doesn't seem to be an explicit time range in the query or using a time range parameter, so it might be implicitly getting 24h as the time range.  it should probably be using the time range parameter?

Reply:
These "Host Config" metrics are supposed to show the current (latest) status only, so the time range shouldn't apply here.